datashield/src/main/java/org/molgenis/datashield/DataController.java

```diff
@@ -89,8 +93,9 @@ public void load(@PathVariable String entityTypeId, @PathVariable String assignS
 
   @PostMapping(value = "/execute", consumes = TEXT_PLAIN_VALUE, produces = APPLICATION_JSON_VALUE)
   public CompletableFuture<ResponseEntity<Object>> execute(
-      @RequestBody String cmd, @RequestParam(defaultValue = "false") boolean async) {
-    CompletableFuture<REXP> result = datashieldSession.schedule(cmd);
+      @RequestBody String expression, @RequestParam(defaultValue = "false") boolean async) {
+    String rewrittenExpression = expressionRewriter.rewriteAggregate(expression);
```

Comment:
We'll need to handle the exception so we don't throw a 500 when the expression is incorrect